README.md

```diff
@@ -1,6 +1,7 @@
 # Juttle Gmail Adapter
 
-Gmail adapter for juttle
+GMail adapter for the [Juttle data flow
```

Comment:
Consider using either `Gmail` everywhere instead of `GMail` since that is how it shows up on google search.